mobile/lib/models/place_details.dart

```diff
@@ -15,33 +14,24 @@ part 'place_details.g.dart';
 
 @JsonSerializable(explicitToJson: true)
 class PlaceDetails {
-  String name;
-  String location;
+  String name = '';
+  String location = '';
   String siteId;
   String placeId = const Uuid().v4();
   double latitude;
   double longitude;
 
-  PlaceDetails(this.name, this.location, this.siteId, this.placeId,
-      this.latitude, this.longitude);
+  PlaceDetails(
+      {required this.name,
+      required this.location,
+      required this.siteId,
+      required this.placeId,
+      required this.latitude,
+      required this.longitude});
 
```

Comment:
Modified contructor to use named parameters

Reply:
just curious, what could be the main difference between `place_details` and `site` models? Do they have different attributes?

Reply:
`place_deatils` model is used for google search. `site` model a semi replica of sites